app/controllers/cooks_controller.rb

```diff
@@ -0,0 +1,74 @@
+class CooksController < ApplicationController
+  before_action :set_cook, only: [:show, :edit, :update, :destroy]
+
+  # GET /cooks
+  # GET /cooks.json
+  def index
+    @cooks = Cook.all
+  end
```

Comment:
Do you have a `/cooks` page?

Reply:
Perfect, thanks for spotting this! These comments are all copy-pastes from the posts scaffold. I haven't written any JSON files, so these lines are totally unnecessary. Thank you so much for going through this for me. I'm learning so much from this review process! ⭐️

Reply:
Are you sure you need an `index` action? Can we get rid of this method?